airbyte-integrations/connectors/source-twitter/source_twitter/spec.yaml

```diff
@@ -4,19 +4,19 @@ connectionSpecification:
   title: Twitter Spec
   type: object
   required:
-    - api_key
+    - api_token
     - query
   additionalProperties: true
   properties:
-    api_key:
+    api_token:
```

Comment:
isn't it a backwards incompatible change? I think a major version should be bumped because of this change